catkin_virtualenv/src/catkin_virtualenv/__init__.py

```diff
@@ -20,16 +20,18 @@
 import logging
 import subprocess
 import rosgraph.roslogging
+import typing
+
 
 logger = logging.getLogger(__name__)
 
 
 def configure_logging():
-    rosgraph.roslogging.configure_logging('catkin_virtualenv')
+    rosgraph.roslogging.configure_logging("catkin_virtualenv")
     return logging.getLogger()
 
 
-def run_command(cmd, *args, **kwargs):
+def run_command(cmd: typing.List[str], *args, **kwargs) -> subprocess.CompletedProcess:
```

Comment:
Not really your problem as this already existed, but we really need to move this into `locus_common` or somewhere. I keep seeing the same block of code copied into various packages :facepalm:

Reply:
That is not a bad idea.

Reply:
catkin_virtualenv is a public package